sdks/java/core/src/main/java/org/apache/beam/sdk/util/MergingActiveWindowSet.java

```diff
@@ -144,7 +144,7 @@ public void ensureWindowIsActive(W window) {
     checkState(stateAddressWindows != null,
                              "Cannot ensure window %s is active since it is neither ACTIVE nor NEW",
                              window);
-    if (stateAddressWindows.isEmpty()) {
+    if (stateAddressWindows != null && stateAddressWindows.isEmpty()) {
```

Comment:
Does it help if you configure findbugs to use the checked-in [findbugs-filter.xml](https://github.com/apache/incubator-beam/blob/master/sdks/java/build-tools/src/main/resources/beam/findbugs-filter.xml)? If not, is there some config we can add to that file to properly detect that `Preconditions` will throw if the checked conditional is true?